problem-gson/src/test/java/org/zalando/problem/InsufficientFundsProblem.java

```diff
@@ -0,0 +1,31 @@
+package org.zalando.problem;
+
+import java.net.URI;
+
+import static org.zalando.problem.Status.BAD_REQUEST;
+
+final class InsufficientFundsProblem extends AbstractThrowableProblem {
+
+    static final String TYPE_VALUE = "https://example.org/insufficient-funds";
+    static final URI TYPE = URI.create(TYPE_VALUE);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/whiskeysierra/problem/pullRequest?prid=4476243)